packages/frontend/editor-ui/src/components/ParameterOptions.vue

```diff
@@ -51,13 +51,15 @@ const isHtmlEditor = computed(
 const shouldShowExpressionSelector = computed(
 	() => !props.parameter.noDataExpression && props.showExpressionSelector && !props.isReadOnly,
 );
+const localResolveCtx = inject(ExpressionLocalResolveContextSymbol, undefined);
+const isInEmbeddedNdv = computed(() => localResolveCtx?.value !== undefined);
 
 const canBeOpenedInFocusPanel = computed(
 	() =>
 		!props.parameter.isNodeSetting &&
 		!props.isReadOnly &&
 		!props.isContentOverridden &&
-		activeNode.value && // checking that it's inside ndv
+		(activeNode.value || isInEmbeddedNdv.value) && // checking that it's inside ndv
```

Comment:
This makes it possible to focus the parameter from within the focus panel

<img width="746" height="537" alt="Image" src="https://github.com/user-attachments/assets/04e54d13-fee8-4bbb-a0dc-3033342ded1c" />